vrc-get-gui/alcom.desktop

```
@@ -0,0 +1,11 @@
+[Desktop Entry]
+Categories={{categories}}
+{{#if comment}}
+Comment={{comment}}
+{{/if}}
+Exec={{exec}}
```

```suggestion
Exec={{exec}} %u
```

without %u, nothing is work.